src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
hmm, what i learned from security folks is that all consequent `Math.random` calls are predictable, which means no extra entropy. So it doesn't really give you more bytes. 

That's to say, when you compare two such arrays that were generated in this way, if the first byte of the arrays equals, all the remaining equal to each other as well.